### PR TITLE
update the demo to use Mobilenet v2

### PR DESCRIPTION
**Pull request by pyu10055**
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-converter/145)

### CODE REVIEW

**Comment by ManrajGrover** [comment]
Also, sharing output screens.

MobilenetV1:
screen shot 2018-06-03 at 3 08 49 pm

MobilenetV2:
screen shot 2018-06-03 at 3 10 29 pm

I haven't read on architectural changes in v2 but seeing output, it looks like it doesn't make use of softmax. Please correct me if I'm wrong.

**Comment by pyu10055** [comment]
added softmax to the logits output, also updated the readme file. Thanks!

---

Review status: 0 of 2 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-converter/145#-:-LEAvOz35iWgogYF7g4d:b-syxzab)*

**Comment by dsmilkov** [comment]
Nice getting MobileNetV2 supported! Couple comments:

1) I don't see the softmax and the README changes? Perhaps forgot to push?
2) Just curious what a single inference takes in V2 vs V1 in the browser? Can you do quick measurements?

---

Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-converter/145#-:-LEB5TOx2imVg7LT5JVS:b100wr5)*

**Comment by dsmilkov** [comment]
:lgtm_strong:

---

Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-converter/145#-:-LEB6IhC4VzYi6Vli3bo:b-jbsf8z)*

**Comment by pyu10055** [comment]
they are compatible both at around 30ms on my desktop.
